bin/monitor_manifest

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env ruby
```

Comment:
Underscore this file name.

Reply:
Like `_monitor_manifest`?

Reply:
I meant "hypenate" - sigh!

`monitor-manifest`

Reply:
Right!